src/ucp/api/ucp_def.h

```diff
@@ -234,5 +234,15 @@ typedef void (*ucp_send_callback_t)(void *request, ucs_status_t status);
 typedef void (*ucp_tag_recv_callback_t)(void *request, ucs_status_t status,
                                         ucp_tag_recv_info_t *info);
 
+/**
+ * @ingroup UCP_WORKER
+ * @brief UCP worker wakeup events mask.
+ *
+ * The enumeration allows specifying which events are expected on wakeup
+ */
+typedef enum ucp_wakeup_event_types {
+    UCP_WAKEUP_TX_COMPLETION   = UCS_BIT(0), /**< TX-completions */
+    UCP_WAKEUP_RX_AM           = UCS_BIT(1)  /**< RX-completions */
```

Comment:
@yosefe is right. From this PR I cannot understand (without getting into the code and even then it is not obvious) what this event means how it it maps to UCP operations that were invoked a user